lib/haml_lint/linter.rb

```diff
@@ -7,6 +7,9 @@ module HamlLint
   class Linter
     include HamlVisitor
 
+    # Linters can override this to indicate that they support the autocorrect system
+    SUPPORTS_AUTOCORRECT = false
```

Comment:
Might make sense to make this work more like `attr_accessor` so that rather than overriding a constant we can simply add to a linter class implementation:

```ruby
supports_autocorrect true
```

...or similar.